src/codex_plus/request_logger.py

```
@@ -0,0 +1,91 @@
+        # Validate branch name to prevent path traversal
+        if not branch or ".." in branch or "/" in branch:
+            branch = "unknown"
+
+        # Parse JSON with specific error handling
+        try:
+            payload = json.loads(body)
+            logger.info(f"Parsed payload with keys: {list(payload.keys())}")
+        except json.JSONDecodeError as e:
+            logger.warning(f"Invalid JSON in request body: {e}")
+            return
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)